docs/src/managing-packages.md

```diff
@@ -208,6 +208,42 @@ For example, the tracked dependencies can be stored inside of the active project
 The whole directory can be moved and `Pkg` will still be able to find the dependencies
 because their path relative to the active project is preserved even though their absolute path has changed.
 
+### Adding a subdir package
+
+If the package you want to add by URL is not in the root of the repository, then you need to manually pass the `subdir` keyword to `Pkg.add`
+or `PackageSpec`. For instance, to add the `SnoopCompileCore` package in the [SnoopCompile](https://github.com/timholy/SnoopCompile.jl)
+repository:
+
+```julia-repl
+julia> Pkg.add(url="https://github.com/timholy/SnoopCompile.jl.git", subdir="SnoopCompileCore")
+     Cloning git-repo `https://github.com/timholy/SnoopCompile.jl.git`
+    Updating git-repo `https://github.com/timholy/SnoopCompile.jl.git`
+   Resolving package versions...
+  Installing known registries into `~/.julia`
+       Added registry `General` to `~/.julia/registries/General`
+    Updating `~/.julia/environments/v1.6/Project.toml`
+  [e2b509da] + SnoopCompileCore v2.7.0 `https://github.com/timholy/SnoopCompile.jl.git:SnoopCompileCore#master`
+    Updating `~/.julia/environments/v1.6/Manifest.toml`
+  [e2b509da] + SnoopCompileCore v2.7.0 `https://github.com/timholy/SnoopCompile.jl.git:SnoopCompileCore#master`
+  [9e88b42a] + Serialization
+Precompiling project...
+  1 dependency successfully precompiled in 2 seconds
+```
+
+Another way is to use the Pkg REPL with `<repo_url>:<subdir>` format:
+
+```julia-repl
+(@v1.5) pkg> add https://github.com/timholy/SnoopCompile.jl.git:SnoopCompileCore # git HTTPS protocol
```

Comment:
```suggestion
pkg> add https://github.com/timholy/SnoopCompile.jl.git:SnoopCompileCore # git HTTPS protocol
```

I think we usually write it like when the active project does not matter. It will be a bit distracting that this will (and already is) an old julia version otherwise.

Reply:
I try to follow the existing examples whose prefix is `v1.0`, and I use `@v1.5` to indicate that this feature requires at least Julia 1.5.

Removed in 1b1a188

Reply:
Okay, then I think we should maybe remove the v1.0 prefixes.